client/public/src/models/PersonModel.js

```diff
@@ -307,7 +309,7 @@ class PersonModel extends BaseModel {
       parts.givenName=rdfUtils.getFirstValue(contact.givenName);
       name.push(parts.givenName);
     }
-    if( id && id.match('^odr') && contact.middleName ) {
+    if(contact.middleName ) {
```

Comment:
@sbagg you need to merge dev back into this RP.  We don't want to overwrite these changes.